design/api_definition.go

```diff
@@ -22,7 +26,7 @@ var _ = a.API("build-tool-detector", func() {
 	})
 
 	a.Scheme("http")
-	a.Host("localhost:8080")
+	a.Host(getHost())
```

Comment:
Let's not do this please, just use a hardcoded string ""openshift.io" since it doesn't matter anyway :)

Reply:
/hmm, maybe i am missing something? When i hard-code this to "openshift.io" in the swagger editor (https://editor.swagger.io/) and using our hosted swagger.json i get this error `TypeError: Failed to fetch` 

However, if it is set to the same host and port as in the config.yml everything works as expected..

Is there another way to get this to work?

Reply:
This is what I typically use http://swagger.goa.design/?url=github.com%2Ffabric8-services%2Fbuild-tool-detector%2Fdesign#!/default/swagger_swagger_json for committed code. They work with branches too.

However, Could you check if `http://localhost:8099/api/swagger.json` as your import URL in editor.swagger.io ? Ideally the above URL should return the swagger response.

Reply:
Ah right, that wouldn't work since localhost wouldn't be reachable from the external world. My bad :)
Since these are somewhat design files, I usually prefer keeping dynamic code at bay. But I'll consider this and try this out myself when I find some time. OK? :)

Reply:
I thought I could re-open PRs, could you try re-opening this? I wanna give this a second thought when I get a chance for a closer look.